packages/composite-checkout/src/lib/payment-methods/existing-credit-card.js

```diff
@@ -219,18 +221,26 @@ function ButtonContents( { formStatus, total } ) {
 function ExistingCardSummary( { cardholderName, cardExpiry, brand, last4 } ) {
 	const { __ } = useI18n();
 
+	const maskedCardDetails = '****'.concat( last4 );
```

Comment:
Why not translate this too, if it was translated already above? (Also, this line as-is is different because there's no longer any space between the asterisks and the last 4).

Reply:
This line (and the one above) is where I was trying to figure out how to get the masked card number to display "correctly" as `**** 4242` rather than `4242 ****`. It seems something about RTL styling reverses text at word boundaries, but the space character here has different semantics. Old checkout manages to do this correctly but I couldn't figure out how.